.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -89,7 +89,7 @@ public IModel GetEmailModel(string culture = null, bool fallbackToDefaultCulture
 
         public IModel GetURLModel(string culture = null, bool fallbackToDefaultCulture = true)
         {
-            return GetModel<URLModel>(Culture.English, fallbackToDefaultCulture);
+            return GetModel<URLModel>(culture, fallbackToDefaultCulture);

```

Comment:
This change will introduce error. The current behaviour is that any language receives the base implementation (English). After having a customized version for Chinese/Japanese there maybe would be logic here to chose which of the two existing models to return.